code/modules/shuttle/medisim.dm

```diff
@@ -0,0 +1,62 @@
+///These are for the medisim shuttle
+
+/obj/machinery/capture_the_flag/medisim
+	game_area = /area/shuttle/escape
+
+/obj/machinery/capture_the_flag/medisim/spawn_team_member(client/new_team_member)
+	var/mob/living/carbon/human/M = ..()
+	M.remove_all_languages(LANGUAGE_CTF)
+	M.grant_language(/datum/language/monkey, TRUE, TRUE, LANGUAGE_CTF)
+
+//maybe add something here about ending the other CTF game/not preventing that from happening
+
+//late initialize for hooking up the computers on the shuttle
+
+/obj/machinery/capture_the_flag/medisim/reset_the_arena()
+	return //so we don't delete every object on the shuttle. downsides = that simulation room is gonna get gross pretty quick
+
+/obj/machinery/capture_the_flag/medisim/red
+	ctf_gear = /datum/outfit/medisimred
+
+/obj/machinery/capture_the_flag/medisim/blue
+	ctf_gear = /datum/outfit/medisimblue
+
+/obj/item/ctf/red/medisim
+	name = "Redfield Castle Fair Maiden"
+	desc = "Protect your maiden, and capture theirs!"
+	icon = 'icons/obj/plushes.dmi'
+	icon_state = "plushie_nuke"
+	game_area = /area/shuttle/escape
+	movement_type = FLYING //there are chasms!
```

Comment:
If someone's holding this and they fall into the chasm, won't it get deleted?

Reply:
frick, dude! hmm...

Reply:
fixed!

Reply:
but i'm keeping this open so i can change FLYING to FLOATING... not that I need either.